src/lib/input/autofill.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, ElementRef, EventEmitter, Injectable, OnDestroy, Output} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {Subject} from 'rxjs/Subject';
+
+
+/** An event that is emitted when the autofill state of an input changes. */
+export type AutofillEvent = {
```

Comment:
Make everything in here `@docs-private`?

Reply:
Why? I do want people to be able to use these

Reply:
Ack, I had thought there was going to be more iteration after this PR.

Reply:
did add some additional doc comments though